permissions/helpers.py

```diff
@@ -273,6 +275,15 @@ def can_view_reservation(user: User, reservation: Reservation) -> bool:
     )
 
 
+def can_view_reservations(user):
```

Comment:
This is probably too broad permission, but maybe this'll be done later since there is another stub right below?

Reply:
I think we could and probably need to show reservations for all users (authenticated) but just not include the user information. Probably different gql queries for those :thinking:

Reply:
Yeah, probably something like minimalized dto that can be used for reserved dates/times etc.